server/liveblog/blogs/blogs.py

```diff
@@ -51,7 +88,13 @@ def on_create(self, docs):
     def get(self, req, lookup):
         if req is None:
             req = ParsedRequest()
-        return self.backend.get('blogs', req=req, lookup=lookup)
+        docs = super().get(req, lookup)
+        return docs
+
+    def find_one(self, req, **lookup):
+        doc = super().find_one(req, **lookup)
+        get_last_cid(doc)
```

Comment:
For a new liveblog developer like me it's hard to understand why we use the getter `get_last_cid` here without keeping the returned value.
It's also hard to guess the role and the meaning of `cid`. What does it mean ?
Maybe you should rename the function by replacing the `get_last_cid` by  `update_last_cid` ?